### ISSUE TITLE
[REVIEW]: GemGIS - Spatial Data Processing for Geomodeling

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#3709]
**Submitting author:** @AlexanderJuestel (<a href="http://orcid.org/0000-0003-0980-7479">Alexander Jüstel</a>)
**Repository:** <a href="https://github.com/cgre-aachen/gemgis" target ="_blank">https://github.com/cgre-aachen/gemgis</a>
**Version:** v1.0.0
**Editor:** @crvernon
**Reviewer:** @omshinde, @kanishkan91
**Archive:** Pending

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

## Status

[![status](https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348/status.svg)](https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348"><img src="https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348/status.svg)](https://joss.theoj.org/papers/5855a6fc36e3a33c4014b3afa535b348)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/cgre-aachen/gemgis" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@omshinde & @kanishkan91, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @crvernon know.

✨ **Please start on your review when you are able, and be sure to complete your review in the next six weeks, at the very latest** ✨ 

## Review checklist for @omshinde
✨ Important: Please do not use the _Convert to issue_ functionality when working through this checklist, instead, please open any new issues associated with your review <a target="_blank" href="https://github.com/cgre-aachen/gemgis">in the software repository associated with the submission</a>. ✨

### Conflict of interest

- [ ] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [ ] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/cgre-aachen/gemgis">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Contribution and authorship:** Has the submitting author (@AlexanderJuestel) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [ ] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [ ] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

## Review checklist for @kanishkan91
✨ Important: Please do not use the _Convert to issue_ functionality when working through this checklist, instead, please open any new issues associated with your review <a target="_blank" href="https://github.com/cgre-aachen/gemgis">in the software repository associated with the submission</a>. ✨

### Conflict of interest

- [ ] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [ ] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/cgre-aachen/gemgis">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Contribution and authorship:** Has the submitting author (@AlexanderJuestel) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
- [ ] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [ ] **A statement of need:** Does the paper have a section titled 'Statement of Need' that clearly states what problems the software is designed to solve and who the target audience is?
- [ ] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [ ] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [ ] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @omshinde, @kanishkan91 it looks like you're currently assigned to review this paper :tada:.

**:warning: JOSS reduced service mode :warning:**

Due to the challenges of the COVID-19 pandemic, JOSS is currently operating in a "reduced service mode". You can read more about what that means in [our blog post](https://blog.joss.theoj.org/2020/05/reopening-joss).

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by crvernon** [comment]
:wave: @omshinde @kanishkan91 @AlexanderJuestel - the review takes place in this issue.

**Comment by crvernon** [comment]
Also, please don't forget to add a link to this review issue in any issues or pull requests you may generate in the  https://github.com/cgre-aachen/gemgis repository. This will help everyone have a single point of reference.

**Comment by AlexanderJuestel** [comment]
@crvernon, thanks for assisting with this! 
@omshinde, @kanishkan91, I am looking forward to go through this review with you guys!

**Comment by AlexanderJuestel** [comment]
@crvernon is there anything that I can do in the meantime or do we have to wait for @omshinde  and @kanishkan91  to start the review?

**Comment by crvernon** [comment]
Thanks for checking in @AlexanderJuestel - at this point we will wait on the first round of feedback to come in as issues/comments. You can address these as they come up to expedite the process. I'll periodically send out check-in comments that open up lines for us to communicate any blockers to progress.

**Comment by kanishkan91** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03709/joss.03709/10.21105.joss.03709.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03709/joss.03709/10.21105.joss.03709.pdf) :page_facing_up: :point_left:

**Comment by kanishkan91** [comment]
@AlexanderJuestel I've started the review. I had some very minor questions for you related to the repository,
1. Firstly, there are two pull requests that are open [here](https://github.com/cgre-aachen/gemgis/pulls). Do these need to be merged in? Just wanted to confirm. 
2. I looked at the recent commit history. It looked like the recent commits were mostly from yourself and @RichardScottOZ. Could you describe the roles that the other co-authors played in the development of the software repository? 
3. Also, in the pydocs, the main developers from the pydocs file - https://gemgis.readthedocs.io/en/latest/getting_started/authors.html don't all appear as co-authors in the paper. Is this intentional? This being a software paper, ideally all major contributors to software development that are mentioned in the docs should be co-authors. 
4.  The license you use is GNU LESSER GENERAL PUBLIC LICENSE. My understanding is limited here, but a GPL license would impose some copy left restrictions on the users (they would have to ensure that they release anything that uses GemGIS as a freely available open source software package). Is this intentional? I have seen packages use BSD2 in its place which offers more flexibility to the end user. @crvernon Do you have any thoughts on this?

**Comment by AlexanderJuestel** [comment]
@kanishkan91, thanks for getting started on the review. Regarding your comments:

1. I merged one of the PRs but would like to keep the second one open as we may change some of the tutorial notebooks during the review process here in combination with updating the documentation. 

2. @RichardScottOZ is a very active user providing always great feedback on typos and grammatical errors in the docstrings or notebooks. However, he has not played an active role in the development of the software yet. 

3. Good point. As for @RichardScottOz, @Japhiolite provided great feedback on the code and the documentation, mostly offline,  but did not provide any functionality for the software. I would add 'Marius Pischke' and 'Miguel de la Varga' as authors to the documentation and change the status of @RichardsScottOz and @Japhiolite.

**Comment by AlexanderJuestel** [comment]
@kanishkan91 I just fixed the authors in the docs. The current build of the documentation is failing though, so changes are only visible locally right now but let's tackle one thing at a time :)

**Comment by kanishkan91** [comment]
@AlexanderJuestel  Thanks. For my point no 2 from above, could you describe the roles the different co-authors played in the development of the software? This does not need to be a part of the paper or repo but having those descriptions here in the review thread would suffice.

**Comment by AlexanderJuestel** [comment]
@kanishkan91, I am the main developer of the software and added most of the functionality of the package. @Endarthur contributed with helpful tips with respect to the implementation of certain functionality. He basically helped me to make things run. Marius Pischke developed with me functionality within the framework of his master thesis at RWTH Aachen University, which I then implemented in GemGIS as he does not have a Github account. @Leguark is the main developer for the GemPy package (https://github.com/cgre-aachen/gempy). Without his developments, GemGIS would not have a purpose. He also contributed minor things to GemGIS as well. @flohorovicic is/was my supervisor who highly encouraged me to develop GemGIS. @Japhiolite and @RichardScottOz mainly provided feedback on the documentation and notebooks and also provided feedback on other communication channels.

Would this description be sufficient or do you need any more information or statements of the respective persons?

**Comment by kanishkan91** [comment]
@AlexanderJuestel This is sufficient. Thanks.

**Comment by crvernon** [comment]
:mega:  Mid-week rally!  Just checking in to see how things are going.  

:open_hands: @kanishkan91 thanks for kicking off this review!  

 @omshinde how are things going?  The author @AlexanderJuestel has been very responsive if you have any questions.  

Have a great and productive rest of the week!

**Comment by whedon** [comment]
:wave: @omshinde, please update us on how your review is going (this is an automated reminder).

**Comment by whedon** [comment]
:wave: @kanishkan91, please update us on how your review is going (this is an automated reminder).

**Comment by omshinde** [comment]
@crvernon Thanks for the reminder. I just finished my other JOSS review and would be able to progress fast with GemGIS submission now. I am very excited and looking forward.

**Comment by crvernon** [comment]
:mega: Hope all is progressing smoothly. 

:wave: @kanishkan91 and @omshinde could you update me on your progress?  Thanks!

**Comment by omshinde** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
:point_right::page_facing_up: [Download article proof](https://raw.githubusercontent.com/openjournals/joss-papers/joss.03709/joss.03709/10.21105.joss.03709.pdf) :page_facing_up: [View article proof on GitHub](https://github.com/openjournals/joss-papers/blob/joss.03709/joss.03709/10.21105.joss.03709.pdf) :page_facing_up: :point_left:

**Comment by kanishkan91** [comment]
@crvernon I am going through the review. I hope to  complete it by this weekend or latest by  the 6th of October. Let me know what you think.

**Comment by crvernon** [comment]
Sounds great @kanishkan91 - thanks for the update!

**Comment by omshinde** [comment]
Hi @crvernon 
Thanks for the reminder. I have commenced my review and I plan to complete it over the weekend. Kind regards.

**Comment by kanishkan91** [comment]
Hi @crvernon. My apologies. I had to complete some sudden deadlines. I just jumped back to the review today. Will try to complete this week.

**Comment by kanishkan91** [comment]
@AlexanderJuestel  I am going through this review and am adding comments as I go through. I hope to be done by Sunday night. But there is one particular feature that I think would highlight the utility of what you have built much more. Also considering the functionality already built, I think this would be low hanging fruit. I have created an issue already but I add the text from the issue below since I think we should start a discussion on this sooner rather than later.

gemgis is clearly a powerful tool that creates data objects as would be required by GemPy. In particular I like how a user can not only make use of gemgis to create a data object but can also make use of this software to do meaningful post processing of GemPy outputs. But, these functionalities can be very effectively illustrated using a dashboard (I would recommend Dash -https://dash.plotly.com/).

In particular Example 1 can be presented using a dashboard as opposed to the text heavy example as it stands right now. As the repo currently stands it is very much usable but not entirely digestible for novice users (the ones who would most benefit from something like this given that they would find GemPy intimidating.) Having that dashboard would effectively illustrate the utility of this software (which is very high). But more importantly, it would help a novice user understand the functionality much better and would help them effectively use this software.

**Comment by AlexanderJuestel** [comment]
@kanishkan91 thanks for the comments so far! I hope to get back to them at the beginning of next week!

**Comment by crvernon** [comment]
:wave: @kanishkan91 - thanks so much for your thoughtful review and feedback!  I am sure @AlexanderJuestel will find much value in these.  As far as additional features go that do not directly address the requirements of the reviewer checklist, we generally do not require these; however, the author may find that these suggestions strengthen the submission and choose to implement them.  Again, thanks for caring about the quality of your review!  Nice work!

**Comment by crvernon** [comment]
:mega: Mid-week rally!  Looks like @AlexanderJuestel and @kanishkan91 have a good dialogue going and are keeping the review progressing. 

:wave: @omshinde - Just checking in to see how things are going?  Let me know if you have any questions. 

❗ @kanishkan91 and @omshinde - Don't forget to keep you checklist up-to-date as you work through the review.  Thanks!

:clap: Keep up the good work!

**Comment by crvernon** [comment]
:mega: Mid-week rally!  @AlexanderJuestel @kanishkan91 and @omshinde could you provide a short update as to how things are going?   Looks like you are making good progress.  Let's keep driving towards getting things in shape for publication!

:clap: Keep up the good work!

**Comment by crvernon** [comment]
:mega: Mid-week rally!  

@kanishkan91 and @omshinde please provide a short update in this issue thread concerning the status of your review.  Is there anything you are waiting on?  Thanks!

@AlexanderJuestel how are things going on your side of things?  Could you provide a short update here as well?  I would like to keep this moving while we have good momentum. Thanks!

**Comment by AlexanderJuestel** [comment]
@crvernon, thanks for checking in! Currently, I am a little limited in time due to my PhD studies. In addition, we are currently working on a new GemPy release so I wanted to wait for that :)

**Comment by crvernon** [comment]
@kanishkan91 and @omshinde please provide a short update in this issue thread concerning the status of your review.  Is there anything you are waiting on?  Thanks!

**Comment by omshinde** [comment]
Hi @crvernon ! Thanks for the reminder.
I have submitted my initial review (https://github.com/openjournals/joss-reviews/issues/3709#issuecomment-953245623) and waiting for the authors response. Once they are addressed, I would be happy to proceed with the review.

**Comment by AlexanderJuestel** [comment]
@crvernon @omshinde We have some crazy days here before the Christmas Holidays. I will get back to the review as soon as I can!

**Comment by omshinde** [comment]
@AlexanderJuestel 
I understand. Best wishes! :)

**Comment by crvernon** [comment]
Thanks for the updates @omshinde and @AlexanderJuestel !

**Comment by kanishkan91** [comment]
@crvernon Apologies for the late message. I am also waiting on the author to respond to comments. 

@AlexanderJuestel Let me know whenever and I can take a look again (I know its a busy time of year, so no hurry!). By the way its interesting that you mentioned that GemPy is also coming out with a release. This would be an interesting opportunity to see the documentation of the GemPy object documentation in gemgis when its ready.

**Comment by crvernon** [comment]
Sounds good @kanishkan91 thanks for the update!

**Comment by crvernon** [comment]
:wave: @AlexanderJuestel just checking in to see how the responses to the reviewers are going?  Thanks!

**Comment by AlexanderJuestel** [comment]
Same as before @crvernon. I will get back to the requests once I have some more time. Thanks for checking in though

**Comment by crvernon** [comment]
:mega: Mid-week rally!

It would be good to get this one wrapped up soon @AlexanderJuestel so we can free up our reviewers.  How are things looking on your side of things for finishing up the remaining issues?  It looks like both @kanishkan91 and @omshinde are waiting on responses to proceed.

Thanks!

**Comment by AlexanderJuestel** [comment]
Hey everyone,
I totally agree. I am back from vacation next week and will tackle the open issues right away then and hope to solve everything by the end of next weekend

**Comment by AlexanderJuestel** [comment]
@crvernon I still have an issue with the documentation, The fixed will be delayed a little, sorry

**Comment by crvernon** [comment]
@AlexanderJuestel - please provide me with an update of your status and a timeline of when you will be able to complete your revisions.  Thanks!

**Comment by AlexanderJuestel** [comment]
@crvernon Right now, I am not sure when I can complete them. I have issues with the documentation that I am currently not able to fix.

**Comment by AlexanderJuestel** [comment]
@crvernon I will try to port the documentation to another service and hope to make it run that way!

**Comment by crvernon** [comment]
Ok @AlexanderJuestel please let me know if you need assistance with this.

**Comment by crvernon** [comment]
@AlexanderJuestel please let me know the status of your changes.  We should try to wrap this up ASAP.  Thanks!

**Comment by AlexanderJuestel** [comment]
I will give the docs another try @crvernon, so hopefully, they will build now!

**Comment by AlexanderJuestel** [comment]
@crvernon I managed to get my documentation running again, finally. I am trying now to finish the remaining open remarks of @omshinde and @kanishkan91

**Comment by AlexanderJuestel** [comment]
@kanishkan91  I would like to ask you to check the following points off your checklist:

- **Example usage**: Here you can find many examples of using GemGIS: https://gemgis.readthedocs.io/en/latest/getting_started/tutorial/index.html

- **Functionality documentation**: I was just able to build the documentation again. See for example https://readthedocs.org/projects/gemgis/builds/16267667/ 

- **Community Guidelines**: Please see the contributing document for that: https://github.com/cgre-aachen/gemgis/blob/main/CONTRIBUTING.md

**Comment by crvernon** [comment]
:wave: @kanishkan91 and @omshinde please see @AlexanderJuestel comments above and feel free to post any questions.

**Comment by kanishkan91** [comment]
@AlexanderJuestel , @crvernon Thanks. I'l complete this latest by EOD tomorrow (8th March 2022)

**Comment by omshinde** [comment]
Thanks @AlexanderJuestel, @crvernon for the update. I will match with @kanishkan91 and review the updates by March 8th EOD.

**Comment by kanishkan91** [comment]
@AlexanderJuestel I ticked off the items on my list. But I had a couple of follow up questions so that I can close all issues that I opened. 

- Since the tests are all set up now, could you create a codecov badge for your repository , so I can check the the percentage of the code covered by tests? If you've not linked this repo to codecov, it is possible here -https://about.codecov.io/blog/python-code-coverage-using-github-actions-and-codecov/ . With this, you can ignore my comment on the Coverage.py. 
- Could you take a look at my comment about cleaning up the readme? I still think you can take out some of the images (where some of the links are broken) and maybe add an explicit link to the example. 

Other than that, I think this is good to go. @crvernon Let me know what you think.

**Comment by omshinde** [comment]
@whedon generate pdf

**Comment by omshinde** [comment]
@editorialbot generate pdf

**Comment by omshinde** [comment]
@AlexanderJuestel @crvernon 
I am ready with my final review and I have checked off the remaining items. The authors have done a great job to provide neat software and I congratulate them for their contributions. The updated documentation is comprehensive and detailed. Thank you to the authors for their efforts. From my side, I am happy to recommend the work to be published in JOSS.

Thanks, @crvernon for the follow-up and for the opportunity to review the manuscript.

**Comment by crvernon** [comment]
@editorialbot generate pdf

**Comment by crvernon** [comment]
@editorialbot check references

**Comment by AlexanderJuestel** [comment]
@crvernon please let me know if there is anything I need to do prior to the publication

Cheers Alex

**Comment by AlexanderJuestel** [comment]
@omshinde @kanishkan91 Have you heard anything from @crvernon? Or is there anything that I still need to do with respect to the publication?

**Comment by kanishkan91** [comment]
@AlexanderJuestel I haven't heard from @crvernon. But at this point of time, he may get back to you with any edits that he requires. You may be required to make some edits after the JOSS editor looks at it. But I think this is mostly done. Let's see what the reviewer says.

**Comment by crvernon** [comment]
@AlexanderJuestel apologies for the delay. I had a family emergency I had to tend to. I'll finalize this process early next week.

**Comment by AlexanderJuestel** [comment]
@crvernon 
I hope you and your family are doing better! 

I just want to ask you to hold back the publishing of the paper for a little while as I need permission from my new institute to publish the work. It should only take a few days though. However, if you request any minor edits, I can for sure implement them already. Sorry for the inconvenience

**Comment by crvernon** [comment]
@editorialbot generate pdf

**Comment by AlexanderJuestel** [comment]
@crvernon I have finally gotten the go-ahead from my institute. I just had to edit the affiliations a little and one citation. 

Please proceed with the publication! 
Cheers Alex

**Comment by AlexanderJuestel** [comment]
@crvernon any news from your side on the progress of the publication? :)

**Comment by crvernon** [comment]
@editorialbot generate pdf

**Comment by crvernon** [comment]
@editorialbot check references

**Comment by crvernon** [comment]
@AlexanderJuestel  we are almost there!  Next is just setting up the archive for your new release. 

- We want to make sure the archival has the correct metadata that JOSS requires.  This includes **a title that matches the paper title and a correct author list.**  

So here is what we have left to do:
- [ ] Conduct a GitHub release of the current reviewed version of the software you now have on the `main` and archive the reviewed software in Zenodo or a similar service (e.g., figshare, an institutional repository)
- [ ] Check the archival deposit (e.g., in Zenodo) to ensure it has the correct metadata. This includes the title (should match the paper title) and author list. You may also add the authors' ORCID.
- [ ] Please list the DOI of the archived version here

I can then move forward with accepting the submission.

**Comment by AlexanderJuestel** [comment]
Hi @crvernon,

please find the archived version of GemGIS here under this DOI Number: https://doi.org/10.5281/zenodo.6511767

**Comment by crvernon** [comment]
@editorialbot set v1.0.0 as version

**Comment by crvernon** [comment]
@editorialbot set 10.5281/zenodo.6511767 as archive

**Comment by crvernon** [comment]
@AlexanderJuestel - thanks for putting together a really nice software product! Thanks to @omshinde and @kanishkan91 for a constructive and timely review!

I am recommending that your submission be accepted. An EIC will review this shortly and confirm final publication if all goes well.

**Comment by crvernon** [comment]
@editorialbot recommend-accept

**Comment by kyleniemeyer** [comment]
@editorialbot accept

**Comment by AlexanderJuestel** [comment]
@crvernon, thank you for serving as editor for this paper!

@omshinde, @kanishkan91, thank you very much for your reviews and the time you put in it!

We finally did it! 
Alex

**Comment by kyleniemeyer** [comment]
Congratulations @AlexanderJuestel on your article's publication in JOSS!

Many thanks to @omshinde and @kanishkan91 for reviewing this, and @crvernon for editing.
